plugins/mathjax/plugin.js

```diff
@@ -365,10 +365,10 @@
 
 				editor.fire( 'lockSnapshot' );
 
-				buffer.setHtml( value );
+				buffer.setHtml( CKEDITOR.tools.htmlSafeByReview(value, 'Encoded using htmlEncode in setValue') );
```

Comment:
Sounds good! (One more idea, though-- for this one and also the other call in this file-- if the string already passed through `htmlEncode`, would it change any further if we passed it through one more round of sanitization right here for visible safety? Feel free to disregard if you can think of a case in which this might break.)

Reply:
SGTM